src/styles.scss

```diff
@@ -82,6 +82,12 @@ body {
   right: 0;
 }
 
+.absolute-top-right {
```

Comment:
where is this getting used?

Reply:
Think it was left over from an earlier design I had for the cards. Removing.